src/ResourceManagement/CustomerInsights/Microsoft.Azure.Management.CustomerInsights/Generated/Models/ConnectorMappingProperties.cs

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator 1.0.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Azure.Management.CustomerInsights.Models
+{
+    using Azure;
+    using Management;
+    using CustomerInsights;
+    using Rest;
+    using Newtonsoft.Json;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// The connector mapping properties.
+    /// </summary>
+    public partial class ConnectorMappingProperties
+    {
+        /// <summary>
+        /// Initializes a new instance of the ConnectorMappingProperties class.
+        /// </summary>
+        public ConnectorMappingProperties()
+        {
+            Format = new ConnectorMappingFormat();
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the ConnectorMappingProperties class.
+        /// </summary>
+        /// <param name="folderPath">The folder path for the mapping.</param>
+        /// <param name="fileFilter">The file filter for the mapping.</param>
+        /// <param name="hasHeader">If the file contains a header or
+        /// not.</param>
+        public ConnectorMappingProperties(ConnectorMappingErrorManagement errorManagement, ConnectorMappingFormat format, ConnectorMappingAvailability availability, IList<ConnectorMappingStructure> structure, ConnectorMappingCompleteOperation completeOperation, string folderPath = default(string), string fileFilter = default(string), bool? hasHeader = default(bool?))
+        {
+            Format = new ConnectorMappingFormat();
+            FolderPath = folderPath;
+            FileFilter = fileFilter;
+            HasHeader = hasHeader;
+            ErrorManagement = errorManagement;
+            Format = format;
+            Availability = availability;
+            Structure = structure;
+            CompleteOperation = completeOperation;
+        }
+
+        /// <summary>
+        /// Gets or sets the folder path for the mapping.
+        /// </summary>
+        [JsonProperty(PropertyName = "folderPath")]
+        public string FolderPath { get; set; }
+
+        /// <summary>
+        /// Gets or sets the file filter for the mapping.
+        /// </summary>
+        [JsonProperty(PropertyName = "fileFilter")]
+        public string FileFilter { get; set; }
+
+        /// <summary>
+        /// Gets or sets if the file contains a header or not.
+        /// </summary>
+        [JsonProperty(PropertyName = "hasHeader")]
+        public bool? HasHeader { get; set; }
+
+        /// <summary>
+        /// </summary>
```

Comment:
I have gone through the swagger file and added all missing descriptions, and generated SDK code with the same.